### DISCUSSION TITLE
How to use bootstrap JS, Is it not added?

### DISCUSSION THREAD

**Discussion by shubh-mehrotra** [nasirkhan/laravel-starter#429] [Q&A]
- My requirement was to have accordion view, is there any default view for that?
  - Is bootstrap.js added anywhere? how can I use that?

**Comment by nasirkhan** [comment]
Bootstrap js is not added as bootstrap.js but all the features are available. I used CoreUI theme in the backend, which extends the Bootstrap. To use accordion or other bootstrap components I would suggest you to to check the coreui doc https://coreui.io

**Comment by shubh-mehrotra** [comment]
Ok, thank you!

and could you please mention that what is the suggested way to add the custom CSS?

**Reply by shubh-mehrotra** [reply]
@nasirkhan could you please mention it?
